### PR TITLE
Create daniel.md

### PR DESCRIPTION
**Pull request by danielvo**
I'm just trying to follow the recipe

### CODE REVIEW

**Review by xiaoliz0 — APPROVED** [review]
Looks great!

**Review by calocascio — APPROVED** [review]
Yes, it was the correct emoji.

**Review by tonjegul — APPROVED** [review]
💥

File: participants/daniel.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Daniel :neckbeard:
```
Comment (sofievo) [inline participants/daniel.md:1]: 👍

**Review by sofievo — APPROVED** [review]

**Review by ankjel — APPROVED** [review]
🔥

**Review by torben-uio — APPROVED** [review]
Everything OK

**Pull request merged**